samples/artifacts/DataPipelining/ConnectSiddhiApps/README.txt

```diff
@@ -0,0 +1,28 @@
+Purpose:
```

Comment:
Remove README

Reply:
The requested changes are addressed by the commit 6d13673.